lib/classes/Variables.js

```diff
@@ -495,9 +522,18 @@ class Variables {
   }
 
   getValueFromSelf(variableString) {
+    let variable = variableString;
+    // The loaded service is altered during load in ~/lib/classes/Service (see loadServiceFileParam)
+    // Account for these so that user's reference to their file populate properly
+    if (variable === 'self:service.name') {
```

Comment:
Maybe we can add a comment to the `Service` class at the proper location to warn people, that any changes done there that affect the service property, have to be checked with this implementation too?

Reply:
Really good point.  Missed commenting this.  Adding additional comment here too.